tsdb/querier/querier.go

```diff
@@ -208,11 +207,11 @@ func (q *Query) exec(blockStream []tsdb.Block, encoding bool) (i interface{}) {
 		return q.ReturnNILResponse()
 	}
 	// decode the selected range of blocks
-	blocksDecoder := decode.NewBlockDecoding(resultingBlockSlice[0].Type)
+	// blocksDecoder := decode.NewBlockDecoding(resultingBlockSlice[0].Type)
```

Comment:
Why did we comment these out?

Reply:
They were creating errors that's why I commented on them.